conda_rpms/build_rpm_structure.py

```diff
@@ -143,11 +139,29 @@ def create_rpmbuild_for_tag(repo, tag_name, target, config):
     with open(spec_fname, 'r') as fh:
         env_spec = yaml.safe_load(fh).get('env', [])
     create_rpmbuild_for_env(manifest, target, config)
-    pkgs = [pkg for _, pkg in manifest]
+
+    index = conda.fetch.fetch_index(list(set([url for url, pkg in manifest])),
+                                    use_cache=False)
+    resolver = Resolve(index)
+
+    # To sort the distributions must match the format of the keys of the index.
+    # For example, most should look like `http://channel::pkg
+    # However channels on anaconda go by their name rather than their url,
+    #  i.e. `conda-forge::pkg`
+    dists = []
+    for url, pkg in manifest:
+        anaconda_url = 'https://conda.anaconda.org/'
+        if url.startswith(anaconda_url):
+            url = url[len(anaconda_url):]
+        dists.append('::'.join([os.path.dirname(url), pkg]))
```

Comment:
So, if I understand this right ... your influencing the sort here by chopping out the `https://conda.anaconda.org/` from the URL to equalize with items in the manifest that have a channel name rather than a full URL? i.e. `https://conda.anaconda/org/wibble/wobble/pkg` becomes `wibble/wobble::pkg` ? and that makes it a fairer sort against `conda-forge::pkg` ...